src/index.js

```diff
@@ -69,6 +69,10 @@ if (window.CanvasRenderingContext2D) {
     fullAPIReady = loadScript('../core/pdf/PDFNet.js');
   }
 
+  if (state.advanced.disableLogs) {
+    window.CoreControls.disableLogs(true);
```

Comment:
Let's just do `window.CoreControls.disableLogs(getHashParams('disableLogs', false))` instead of storing `disableLogs` in redux since it's not really viewer related and we don't have any actions to change this state. The `exposedSelectors` is a bit misleading since we aren't exposing the selectors in this file... If we'd like to add a getter to the API then we need to change `apis/index.js`